Multiple @NpmPackage annotations on the same class ignored

### Description of the bug
When using ~specific~ multiple  `@NpmPackage` Annotations ~as value of the with `PolymerTemplates` or `Component`~, the `@NpmPackage` Annotations may being ignored if the class is inside a dependency (addon in this case).  
This issue only occurs to me when using the Spring+Boot Starter.   
When using the Jetty based starter it works out of the box.  

~The reason why I think this happens only to certain npm dependencies is that some are working and some don't f.e. "@polymer/..." do works for me. Maybe only packages with prefixes (f.e. `@NpmPackage(value = "@polymer/app-layout", version = "3.1.0")`) work but packages without (`@NpmPackage(value = "apexcharts", version = "3.8.4")`) don't 😕? I'm not sure at all maybe somebody could help me out here?~

<details>
<summary>Webpack fails due to missing dependency</summary>

```
2019-08-23 14:34:35.513  INFO 19796 --- [  restartedMain] dev-updater                              : Updated C:\Users\Johannes\Desktop\my-starter-project\target\frontend\generated-flow-imports.js
2019-08-23 14:34:37.969  INFO 19796 --- [  restartedMain] dev-webpack                              : Starting webpack-dev-server, port: 56908 dir: C:\Users\Johannes\Desktop\my-starter-project
   C:\Program Files\nodejs\node.exe C:\Users\Johannes\Desktop\my-starter-project\node_modules\webpack-dev-server\bin\webpack-dev-server.js --config C:\Users\Johannes\Desktop\my-starter-project\webpack.config.js --port 56908 -d --inline=false
2019-08-23 14:34:40.194  INFO 19796 --- [        webpack] dev-webpack                              : i ｢wds｣: Project is running at http://localhost:56908/webpack-dev-server/
2019-08-23 14:34:40.194  INFO 19796 --- [        webpack] dev-webpack                              : i ｢wds｣: webpack output is served from /
2019-08-23 14:34:40.194  INFO 19796 --- [        webpack] dev-webpack                              : i ｢wds｣: Content not from webpack is served from C:\Users\Johannes\Desktop\my-starter-project\target\classes\META-INF\VAADIN, src/main/webapp
2019-08-23 14:34:45.554  INFO 19796 --- [        webpack] dev-webpack                              :          Serving the 'stats.json' file dynamically.
2019-08-23 14:34:45.629  INFO 19796 --- [        webpack] dev-webpack                              : × ｢wdm｣: Hash: b4981388a366c4a26ffe
2019-08-23 14:34:45.629  INFO 19796 --- [        webpack] dev-webpack                              : Version: webpack 4.30.0
2019-08-23 14:34:45.629  INFO 19796 --- [        webpack] dev-webpack                              : Time: 5399ms
2019-08-23 14:34:45.629  INFO 19796 --- [        webpack] dev-webpack                              : Built at: 2019-08-23 14:34:45
2019-08-23 14:34:45.629  INFO 19796 --- [        webpack] dev-webpack                              :                                                                  Asset       Size      Chunks                    Chunk Names
2019-08-23 14:34:45.629  INFO 19796 --- [        webpack] dev-webpack                              :                      build/vaadin-bundle-4137a6629b456b7a593f.cache.js   3.53 MiB      bundle  [emitted]  [big]  bundle
2019-08-23 14:34:45.629  INFO 19796 --- [        webpack] dev-webpack                              :                  build/vaadin-bundle.es5-0a353625349d43ec60d8.cache.js   3.92 MiB  bundle.es5  [emitted]  [big]  bundle.es5
2019-08-23 14:34:45.629  INFO 19796 --- [        webpack] dev-webpack                              :                                       build\webcomponentsjs\LICENSE.md   1.52 KiB              [emitted]         
2019-08-23 14:34:45.629  INFO 19796 --- [        webpack] dev-webpack                              :                                        build\webcomponentsjs\README.md   10.9 KiB              [emitted]         
2019-08-23 14:34:45.629  INFO 19796 --- [        webpack] dev-webpack                              :                      build\webcomponentsjs\bundles\webcomponents-ce.js   16.7 KiB              [emitted]         
2019-08-23 14:34:45.629  INFO 19796 --- [        webpack] dev-webpack                              :                  build\webcomponentsjs\bundles\webcomponents-ce.js.map    112 KiB              [emitted]         
2019-08-23 14:34:45.629  INFO 19796 --- [        webpack] dev-webpack                              :                build\webcomponentsjs\bundles\webcomponents-sd-ce-pf.js    108 KiB              [emitted]         
2019-08-23 14:34:45.629  INFO 19796 --- [        webpack] dev-webpack                              :            build\webcomponentsjs\bundles\webcomponents-sd-ce-pf.js.map    653 KiB              [emitted]         
2019-08-23 14:34:45.629  INFO 19796 --- [        webpack] dev-webpack                              :                   build\webcomponentsjs\bundles\webcomponents-sd-ce.js   80.9 KiB              [emitted]         
2019-08-23 14:34:45.630  INFO 19796 --- [        webpack] dev-webpack                              :               build\webcomponentsjs\bundles\webcomponents-sd-ce.js.map    527 KiB              [emitted]         
2019-08-23 14:34:45.630  INFO 19796 --- [        webpack] dev-webpack                              :                      build\webcomponentsjs\bundles\webcomponents-sd.js   65.1 KiB              [emitted]         
2019-08-23 14:34:45.630  INFO 19796 --- [        webpack] dev-webpack                              :                  build\webcomponentsjs\bundles\webcomponents-sd.js.map    415 KiB              [emitted]         
2019-08-23 14:34:45.630  INFO 19796 --- [        webpack] dev-webpack                              :                   build\webcomponentsjs\custom-elements-es5-adapter.js  924 bytes              [emitted]         
2019-08-23 14:34:45.630  INFO 19796 --- [        webpack] dev-webpack                              : build\webcomponentsjs\entrypoints\custom-elements-es5-adapter-index.js  656 bytes              [emitted]         
2019-08-23 14:34:45.630  INFO 19796 --- [        webpack] dev-webpack                              :        build\webcomponentsjs\entrypoints\webcomponents-bundle-index.js   1.66 KiB              [emitted]         
2019-08-23 14:34:45.630  INFO 19796 --- [        webpack] dev-webpack                              :            build\webcomponentsjs\entrypoints\webcomponents-ce-index.js  672 bytes              [emitted]         
2019-08-23 14:34:45.630  INFO 19796 --- [        webpack] dev-webpack                              :         build\webcomponentsjs\entrypoints\webcomponents-sd-ce-index.js  851 bytes              [emitted]         
2019-08-23 14:34:45.630  INFO 19796 --- [        webpack] dev-webpack                              :      build\webcomponentsjs\entrypoints\webcomponents-sd-ce-pf-index.js   1.19 KiB              [emitted]         
2019-08-23 14:34:45.630  INFO 19796 --- [        webpack] dev-webpack                              :            build\webcomponentsjs\entrypoints\webcomponents-sd-index.js  761 bytes              [emitted]         
2019-08-23 14:34:45.630  INFO 19796 --- [        webpack] dev-webpack                              :                                     build\webcomponentsjs\package.json      3 KiB              [emitted]         
2019-08-23 14:34:45.630  INFO 19796 --- [        webpack] dev-webpack                              :                          build\webcomponentsjs\webcomponents-bundle.js    109 KiB              [emitted]         
2019-08-23 14:34:45.630  INFO 19796 --- [        webpack] dev-webpack                              :                      build\webcomponentsjs\webcomponents-bundle.js.map    655 KiB              [emitted]         
2019-08-23 14:34:45.630  INFO 19796 --- [        webpack] dev-webpack                              :                          build\webcomponentsjs\webcomponents-loader.js   6.13 KiB              [emitted]         
2019-08-23 14:34:45.630  INFO 19796 --- [        webpack] dev-webpack                              : Entrypoint bundle [big] = build/vaadin-bundle-4137a6629b456b7a593f.cache.js
2019-08-23 14:34:45.630  INFO 19796 --- [        webpack] dev-webpack                              : Entrypoint bundle.es5 [big] = build/vaadin-bundle.es5-0a353625349d43ec60d8.cache.js
2019-08-23 14:34:45.630  INFO 19796 --- [        webpack] dev-webpack                              : [../node_modules/@polymer/app-layout/app-drawer-layout/app-drawer-layout.js] 9.42 KiB {bundle.es5} [built]
2019-08-23 14:34:45.630  INFO 19796 --- [        webpack] dev-webpack                              : [../node_modules/@polymer/app-layout/app-drawer-layout/app-drawer-layout.js] 7.75 KiB {bundle} [built]
2019-08-23 14:34:45.630  INFO 19796 --- [        webpack] dev-webpack                              : [../node_modules/@polymer/app-layout/app-drawer/app-drawer.js] 21.5 KiB {bundle.es5} [built]
2019-08-23 14:34:45.631  INFO 19796 --- [        webpack] dev-webpack                              : [../node_modules/@polymer/app-layout/app-drawer/app-drawer.js] 18.2 KiB {bundle} [built]
2019-08-23 14:34:45.631  INFO 19796 --- [        webpack] dev-webpack                              : [../node_modules/@polymer/iron-collapse/iron-collapse.js] 7.99 KiB {bundle.es5} [built]
2019-08-23 14:34:45.631  INFO 19796 --- [        webpack] dev-webpack                              : [../node_modules/@polymer/iron-collapse/iron-collapse.js] 7.58 KiB {bundle} [built]
2019-08-23 14:34:45.631  INFO 19796 --- [        webpack] dev-webpack                              : [../node_modules/@polymer/iron-icon/iron-icon.js] 5.88 KiB {bundle.es5} [built]
2019-08-23 14:34:45.631  INFO 19796 --- [        webpack] dev-webpack                              : [../node_modules/@polymer/iron-icon/iron-icon.js] 5.56 KiB {bundle} [built]
2019-08-23 14:34:45.631  INFO 19796 --- [        webpack] dev-webpack                              : [../node_modules/@polymer/paper-badge/paper-badge.js] 9.21 KiB {bundle.es5} [built]
2019-08-23 14:34:45.631  INFO 19796 --- [        webpack] dev-webpack                              : [../node_modules/@polymer/paper-badge/paper-badge.js] 7.63 KiB {bundle} [built]
2019-08-23 14:34:45.631  INFO 19796 --- [        webpack] dev-webpack                              : [../node_modules/@polymer/paper-ripple/paper-ripple.js] 18.8 KiB {bundle.es5} [built]
2019-08-23 14:34:45.631  INFO 19796 --- [        webpack] dev-webpack                              : [../node_modules/@polymer/paper-ripple/paper-ripple.js] 18.2 KiB {bundle} [built]
2019-08-23 14:34:45.631  INFO 19796 --- [        webpack] dev-webpack                              : [../node_modules/@vaadin/flow-frontend/com/github/appreciated/apexcharts/apexcharts-wrapper-styles.css] 220 bytes {bundle} {bundle.es5} [built]
2019-08-23 14:34:45.631  INFO 19796 --- [        webpack] dev-webpack                              : [../target/frontend/generated-flow-imports.js] 3.65 KiB {bundle.es5} [built]
2019-08-23 14:34:45.631  INFO 19796 --- [        webpack] dev-webpack                              : [../target/frontend/generated-flow-imports.js] 3.62 KiB {bundle} [built]
2019-08-23 14:34:45.631  INFO 19796 --- [        webpack] dev-webpack                              :     + 302 hidden modules
2019-08-23 14:34:45.631  INFO 19796 --- [        webpack] dev-webpack                              : 
2019-08-23 14:34:45.631 ERROR 19796 --- [        webpack] dev-webpack                              : ERROR in ../target/frontend/generated-flow-imports.js
2019-08-23 14:34:45.631 ERROR 19796 --- [        webpack] dev-webpack                              : Module not found: Error: Can't resolve 'apexcharts/apexcharts.js' in 'C:\Users\Johannes\Desktop\my-starter-project\target\frontend'
2019-08-23 14:34:45.631 ERROR 19796 --- [        webpack] dev-webpack                              :  @ ../target/frontend/generated-flow-imports.js 43:0-34
2019-08-23 14:34:45.631 ERROR 19796 --- [        webpack] dev-webpack                              : 
2019-08-23 14:34:45.631 ERROR 19796 --- [        webpack] dev-webpack                              : ERROR in ../target/frontend/generated-flow-imports.js
2019-08-23 14:34:45.631 ERROR 19796 --- [        webpack] dev-webpack                              : Module not found: Error: Can't resolve 'apexcharts/apexcharts.js' in 'C:\Users\Johannes\Desktop\my-starter-project\target\frontend'
2019-08-23 14:34:45.631 ERROR 19796 --- [        webpack] dev-webpack                              :  @ ../target/frontend/generated-flow-imports.js 43:0-34
2019-08-23 14:34:45.632 ERROR 19796 --- [        webpack] dev-webpack                              : 
2019-08-23 14:34:45.632 ERROR 19796 --- [        webpack] dev-webpack                              : ERROR in ../node_modules/@vaadin/flow-frontend/com/github/appreciated/apexcharts/apexcharts-wrapper.js
2019-08-23 14:34:45.632 ERROR 19796 --- [        webpack] dev-webpack                              : Module not found: Error: Can't resolve 'apexcharts' in 'C:\Users\Johannes\Desktop\my-starter-project\node_modules\@vaadin\flow-frontend\com\github\appreciated\apexcharts'
2019-08-23 14:34:45.632 ERROR 19796 --- [        webpack] dev-webpack                              :  @ ../node_modules/@vaadin/flow-frontend/com/github/appreciated/apexcharts/apexcharts-wrapper.js 22:0-36 44:32-42
2019-08-23 14:34:45.632 ERROR 19796 --- [        webpack] dev-webpack                              :  @ ../target/frontend/generated-flow-imports.js
2019-08-23 14:34:45.632 ERROR 19796 --- [        webpack] dev-webpack                              : 
2019-08-23 14:34:45.632 ERROR 19796 --- [        webpack] dev-webpack                              : ERROR in ../node_modules/@vaadin/flow-frontend/com/github/appreciated/apexcharts/apexcharts-wrapper.js
2019-08-23 14:34:45.632 ERROR 19796 --- [        webpack] dev-webpack                              : Module not found: Error: Can't resolve 'apexcharts' in 'C:\Users\Johannes\Desktop\my-starter-project\node_modules\@vaadin\flow-frontend\com\github\appreciated\apexcharts'
2019-08-23 14:34:45.632 ERROR 19796 --- [        webpack] dev-webpack                              :  @ ../node_modules/@vaadin/flow-frontend/com/github/appreciated/apexcharts/apexcharts-wrapper.js 4:0-36 123:30-40
2019-08-23 14:34:45.632 ERROR 19796 --- [        webpack] dev-webpack                              :  @ ../target/frontend/generated-flow-imports.js
2019-08-23 14:34:45.632 ERROR 19796 --- [        webpack] dev-webpack                              : 
2019-08-23 14:34:45.632 ERROR 19796 --- [        webpack] dev-webpack                              : ERROR in ../target/frontend/generated-flow-imports.js
2019-08-23 14:34:45.632 ERROR 19796 --- [        webpack] dev-webpack                              : Module not found: Error: Can't resolve 'onecolor/one-color.js' in 'C:\Users\Johannes\Desktop\my-starter-project\target\frontend'
2019-08-23 14:34:45.632 ERROR 19796 --- [        webpack] dev-webpack                              :  @ ../target/frontend/generated-flow-imports.js 51:0-31
2019-08-23 14:34:45.632 ERROR 19796 --- [        webpack] dev-webpack                              : 
2019-08-23 14:34:45.632 ERROR 19796 --- [        webpack] dev-webpack                              : ERROR in ../target/frontend/generated-flow-imports.js
2019-08-23 14:34:45.632 ERROR 19796 --- [        webpack] dev-webpack                              : Module not found: Error: Can't resolve 'onecolor/one-color.js' in 'C:\Users\Johannes\Desktop\my-starter-project\target\frontend'
2019-08-23 14:34:45.632 ERROR 19796 --- [        webpack] dev-webpack                              :  @ ../target/frontend/generated-flow-imports.js 51:0-31
2019-08-23 14:34:45.632 ERROR 19796 --- [        webpack] dev-webpack                              : 
2019-08-23 14:34:45.633 ERROR 19796 --- [        webpack] dev-webpack                              : ERROR in ../node_modules/@vaadin/flow-frontend/com/github/appreciated/apexcharts/apexcharts-wrapper.js
2019-08-23 14:34:45.633 ERROR 19796 --- [        webpack] dev-webpack                              : Module not found: Error: Can't resolve 'onecolor' in 'C:\Users\Johannes\Desktop\my-starter-project\node_modules\@vaadin\flow-frontend\com\github\appreciated\apexcharts'
2019-08-23 14:34:45.633 ERROR 19796 --- [        webpack] dev-webpack                              :  @ ../node_modules/@vaadin/flow-frontend/com/github/appreciated/apexcharts/apexcharts-wrapper.js 40:19-38
2019-08-23 14:34:45.633 ERROR 19796 --- [        webpack] dev-webpack                              :  @ ../target/frontend/generated-flow-imports.js
2019-08-23 14:34:45.633 ERROR 19796 --- [        webpack] dev-webpack                              : 
2019-08-23 14:34:45.633 ERROR 19796 --- [        webpack] dev-webpack                              : ERROR in ../node_modules/@vaadin/flow-frontend/com/github/appreciated/apexcharts/apexcharts-wrapper.js
2019-08-23 14:34:45.633 ERROR 19796 --- [        webpack] dev-webpack                              : Module not found: Error: Can't resolve 'onecolor' in 'C:\Users\Johannes\Desktop\my-starter-project\node_modules\@vaadin\flow-frontend\com\github\appreciated\apexcharts'
2019-08-23 14:34:45.633 ERROR 19796 --- [        webpack] dev-webpack                              :  @ ../node_modules/@vaadin/flow-frontend/com/github/appreciated/apexcharts/apexcharts-wrapper.js 119:17-36
2019-08-23 14:34:45.633 ERROR 19796 --- [        webpack] dev-webpack                              :  @ ../target/frontend/generated-flow-imports.js
2019-08-23 14:34:45.633 ERROR 19796 --- [        webpack] dev-webpack                              : i ｢wdm｣: Failed to compile.
2019-08-23 14:34:45.633 ERROR 19796 --- [        webpack] dev-webpack                              : 
------------------ Frontend compilation failed. -----------------
```
</details>

### Minimal reproducible example
```
@NpmPackage(value = "apexcharts", version = "3.8.4")
@NpmPackage(value = "onecolor", version = "3.0.5")
....
public class ApexCharts extends PolymerTemplate  {


}
```

### dev-updater output

<details>
<summary>output dev-updater</summary>

```
2019-08-23 11:28:29.098  INFO 1080 --- [  restartedMain] c.github.appreciated.spring.Application  : Starting Application on DESKTOP-2O2IRF7 with PID 1080 (C:\Users\Johannes\Desktop\my-starter-project\target\classes started by Johannes in C:\Users\Johannes\Desktop\my-starter-project)
2019-08-23 11:28:29.100  INFO 1080 --- [  restartedMain] c.github.appreciated.spring.Application  : No active profile set, falling back to default profiles: default
2019-08-23 11:28:29.134  INFO 1080 --- [  restartedMain] .e.DevToolsPropertyDefaultsPostProcessor : Devtools property defaults active! Set 'spring.devtools.add-properties' to 'false' to disable
2019-08-23 11:28:29.134  INFO 1080 --- [  restartedMain] .e.DevToolsPropertyDefaultsPostProcessor : For additional web related logging consider setting the 'logging.level.web' property to 'DEBUG'
2019-08-23 11:28:29.875  INFO 1080 --- [  restartedMain] o.s.b.w.embedded.tomcat.TomcatWebServer  : Tomcat initialized with port(s): 8080 (http)
2019-08-23 11:28:29.889  INFO 1080 --- [  restartedMain] o.apache.catalina.core.StandardService   : Starting service [Tomcat]
2019-08-23 11:28:29.889  INFO 1080 --- [  restartedMain] org.apache.catalina.core.StandardEngine  : Starting Servlet engine: [Apache Tomcat/9.0.22]
2019-08-23 11:28:29.973  INFO 1080 --- [  restartedMain] o.a.c.c.C.[Tomcat].[localhost].[/]       : Initializing Spring embedded WebApplicationContext
2019-08-23 11:28:29.973  INFO 1080 --- [  restartedMain] o.s.web.context.ContextLoader            : Root WebApplicationContext: initialization completed in 839 ms
2019-08-23 11:28:48.338  INFO 1080 --- [  restartedMain] c.v.f.server.startup.DevModeInitializer  : Starting dev-mode updaters in C:\Users\Johannes\Desktop\my-starter-project folder.
2019-08-23 11:28:48.342  INFO 1080 --- [  restartedMain] dev-updater                              : Scanning classes to find frontend configurations and dependencies...
2019-08-23 11:28:48.941  INFO 1080 --- [  restartedMain] dev-updater                              : Visited 790 classes. Took 598 ms.
2019-08-23 11:28:48.947  INFO 1080 --- [  restartedMain] dev-updater                              : Added "@vaadin/vaadin-crud": "1.0.5" line.
2019-08-23 11:28:48.947  INFO 1080 --- [  restartedMain] dev-updater                              : Added "@vaadin/vaadin-icons": "4.3.1" line.
2019-08-23 11:28:48.947  INFO 1080 --- [  restartedMain] dev-updater                              : Added "@vaadin/vaadin-grid": "5.4.6" line.
2019-08-23 11:28:48.947  INFO 1080 --- [  restartedMain] dev-updater                              : Added "@polymer/iron-collapse": "3.0.1" line.
2019-08-23 11:28:48.947  INFO 1080 --- [  restartedMain] dev-updater                              : Added "@vaadin/vaadin-split-layout": "4.1.1" line.
2019-08-23 11:28:48.947  INFO 1080 --- [  restartedMain] dev-updater                              : Added "@vaadin/vaadin-cookie-consent": "1.1.1" line.
2019-08-23 11:28:48.947  INFO 1080 --- [  restartedMain] dev-updater                              : Added "@vaadin/vaadin-combo-box": "5.0.6" line.
2019-08-23 11:28:48.947  INFO 1080 --- [  restartedMain] dev-updater                              : Added "@vaadin/vaadin-core-shrinkwrap": "14.0.1" line.
2019-08-23 11:28:48.947  INFO 1080 --- [  restartedMain] dev-updater                              : Added "@vaadin/vaadin-upload": "4.2.2" line.
2019-08-23 11:28:48.947  INFO 1080 --- [  restartedMain] dev-updater                              : Added "@vaadin/vaadin-dialog": "2.2.1" line.
2019-08-23 11:28:48.947  INFO 1080 --- [  restartedMain] dev-updater                              : Added "@vaadin/vaadin-select": "2.1.5" line.
2019-08-23 11:28:48.947  INFO 1080 --- [  restartedMain] dev-updater                              : Added "@vaadin/vaadin-app-layout": "2.0.2" line.
2019-08-23 11:28:48.947  INFO 1080 --- [  restartedMain] dev-updater                              : Added "@vaadin/vaadin-item": "2.1.0" line.
2019-08-23 11:28:48.947  INFO 1080 --- [  restartedMain] dev-updater                              : Added "@polymer/app-layout": "3.1.0" line.
2019-08-23 11:28:48.947  INFO 1080 --- [  restartedMain] dev-updater                              : Added "@vaadin/vaadin-board": "2.1.0" line.
2019-08-23 11:28:48.947  INFO 1080 --- [  restartedMain] dev-updater                              : Added "@vaadin/vaadin-charts": "6.2.3" line.
2019-08-23 11:28:48.947  INFO 1080 --- [  restartedMain] dev-updater                              : Added "@vaadin/vaadin-notification": "1.4.0" line.
2019-08-23 11:28:48.947  INFO 1080 --- [  restartedMain] dev-updater                              : Added "@vaadin/vaadin-grid-pro": "2.0.3" line.
2019-08-23 11:28:48.947  INFO 1080 --- [  restartedMain] dev-updater                              : Added "@vaadin/vaadin-progress-bar": "1.1.2" line.
2019-08-23 11:28:48.947  INFO 1080 --- [  restartedMain] dev-updater                              : Added "@vaadin/vaadin-shrinkwrap": "14.0.1" line.
2019-08-23 11:28:48.947  INFO 1080 --- [  restartedMain] dev-updater                              : Added "@vaadin/vaadin-ordered-layout": "1.1.0" line.
2019-08-23 11:28:48.947  INFO 1080 --- [  restartedMain] dev-updater                              : Added "@vaadin/vaadin-login": "1.0.1" line.
2019-08-23 11:28:48.947  INFO 1080 --- [  restartedMain] dev-updater                              : Added "@vaadin/vaadin-button": "2.2.1" line.
2019-08-23 11:28:48.947  INFO 1080 --- [  restartedMain] dev-updater                              : Added "@vaadin/vaadin-date-picker": "4.0.3" line.
2019-08-23 11:28:48.947  INFO 1080 --- [  restartedMain] dev-updater                              : Added "@vaadin/vaadin-text-field": "2.4.8" line.
2019-08-23 11:28:48.947  INFO 1080 --- [  restartedMain] dev-updater                              : Added "@vaadin/vaadin-menu-bar": "1.0.3" line.
2019-08-23 11:28:48.948  INFO 1080 --- [  restartedMain] dev-updater                              : Added "@vaadin/vaadin-form-layout": "2.1.4" line.
2019-08-23 11:28:48.948  INFO 1080 --- [  restartedMain] dev-updater                              : Added "@polymer/paper-tabs": "3.1.0" line.
2019-08-23 11:28:48.948  INFO 1080 --- [  restartedMain] dev-updater                              : Added "@vaadin/vaadin-accordion": "1.0.1" line.
2019-08-23 11:28:48.948  INFO 1080 --- [  restartedMain] dev-updater                              : Added "@vaadin/vaadin-confirm-dialog": "1.1.4" line.
2019-08-23 11:28:48.948  INFO 1080 --- [  restartedMain] dev-updater                              : Added "@polymer/iron-list": "3.0.2" line.
2019-08-23 11:28:48.948  INFO 1080 --- [  restartedMain] dev-updater                              : Added "@vaadin/vaadin-list-box": "1.1.1" line.
2019-08-23 11:28:48.948  INFO 1080 --- [  restartedMain] dev-updater                              : Added "@polymer/paper-badge": "3.0.1" line.
2019-08-23 11:28:48.948  INFO 1080 --- [  restartedMain] dev-updater                              : Added "@vaadin/vaadin-details": "1.0.1" line.
2019-08-23 11:28:48.948  INFO 1080 --- [  restartedMain] dev-updater                              : Added "@vaadin/vaadin-checkbox": "2.2.10" line.
2019-08-23 11:28:48.948  INFO 1080 --- [  restartedMain] dev-updater                              : Added "@polymer/iron-icon": "3.0.1" line.
2019-08-23 11:28:48.948  INFO 1080 --- [  restartedMain] dev-updater                              : Added "@polymer/paper-ripple": "3.0.1" line.
2019-08-23 11:28:48.948  INFO 1080 --- [  restartedMain] dev-updater                              : Added "@vaadin/vaadin-time-picker": "2.0.2" line.
2019-08-23 11:28:48.948  INFO 1080 --- [  restartedMain] dev-updater                              : Added "@vaadin/vaadin-context-menu": "4.3.12" line.
2019-08-23 11:28:48.948  INFO 1080 --- [  restartedMain] dev-updater                              : Added "@vaadin/vaadin-tabs": "3.0.4" line.
2019-08-23 11:28:48.948  INFO 1080 --- [  restartedMain] dev-updater                              : Added "@vaadin/vaadin-radio-button": "1.2.3" line.
2019-08-23 11:28:48.948  INFO 1080 --- [  restartedMain] dev-updater                              : Added "@vaadin/vaadin-lumo-styles": "1.5.0" line.
2019-08-23 11:28:48.948  INFO 1080 --- [  restartedMain] dev-updater                              : Added "@vaadin/vaadin-material-styles": "1.2.3" line.
2019-08-23 11:28:48.948  INFO 1080 --- [  restartedMain] dev-updater                              : Added "@vaadin/vaadin-rich-text-editor": "1.0.4" line.
2019-08-23 11:28:48.956  INFO 1080 --- [  restartedMain] dev-updater                              : Updated npm C:\Users\Johannes\Desktop\my-starter-project\target\frontend\package.json.
2019-08-23 11:28:48.959  INFO 1080 --- [  restartedMain] dev-updater                              : Running `npm install` ...

> @vaadin/vaadin-usage-statistics@2.0.4 postinstall C:\Users\Johannes\Desktop\my-starter-project\node_modules\@vaadin\vaadin-core-shrinkwrap\node_modules\@vaadin\vaadin-usage-statistics
> node check.js


    Vaadin collects development time usage statistics to improve this product. To opt-out, either run:
    npm explore @vaadin/vaadin-usage-statistics -- npm run disable
    to store disable statistics for the machine, or add
    "vaadin": { "disableUsageStatistics": true }
    to the project package.json and re-run npm install to disable statistics for the project.

    For more details, see https://github.com/vaadin/vaadin-usage-statistics
  

> @vaadin/vaadin-usage-statistics@2.0.4 postinstall C:\Users\Johannes\Desktop\my-starter-project\node_modules\@vaadin\vaadin-shrinkwrap\node_modules\@vaadin\vaadin-usage-statistics
> node check.js


    Vaadin collects development time usage statistics to improve this product. To opt-out, either run:
    npm explore @vaadin/vaadin-usage-statistics -- npm run disable
    to store disable statistics for the machine, or add
    "vaadin": { "disableUsageStatistics": true }
    to the project package.json and re-run npm install to disable statistics for the project.

    For more details, see https://github.com/vaadin/vaadin-usage-statistics
  

> @vaadin/vaadin-usage-statistics@2.0.9 postinstall C:\Users\Johannes\Desktop\my-starter-project\node_modules\@vaadin\vaadin-usage-statistics
> node check.js


    Vaadin collects development time usage statistics to improve this product. To opt-out, either run:
    npm explore @vaadin/vaadin-usage-statistics -- npm run disable
    to store disable statistics for the machine, or add
    "vaadin": { "disableUsageStatistics": true }
    to the project package.json and re-run npm install to disable statistics for the project.

    For more details, see https://github.com/vaadin/vaadin-usage-statistics
  

> core-js@2.6.9 postinstall C:\Users\Johannes\Desktop\my-starter-project\node_modules\core-js
> node scripts/postinstall || echo "ignore"

npm notice created a lockfile as package-lock.json. You should commit this file.
npm WARN no-name@ No repository field.
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@1.2.9 (node_modules\fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.2.9: wanted {"os":"darwin","arch":"any"} (current: {"os":"win32","arch":"x64"})

added 901 packages from 394 contributors and audited 22801 packages in 35.368s
found 0 vulnerabilities

2019-08-23 11:29:25.652  INFO 1080 --- [  restartedMain] dev-updater                              : package.json updated and npm dependencies installed. 
2019-08-23 11:29:25.652  INFO 1080 --- [  restartedMain] dev-updater                              : Copying frontend resources from jar files ...
2019-08-23 11:29:25.716  INFO 1080 --- [  restartedMain] dev-updater                              : Visited 12 jar files. Took 63 ms.
2019-08-23 11:29:25.720  WARN 1080 --- [  restartedMain] dev-updater                              : Use the './' prefix for files in JAR files: 'flow-component-renderer.js', please update your component.
2019-08-23 11:29:25.739  INFO 1080 --- [  restartedMain] dev-updater                              : Updated C:\Users\Johannes\Desktop\my-starter-project\target\frontend\generated-flow-imports.js
2019-08-23 11:29:27.629  INFO 1080 --- [  restartedMain] dev-webpack                              : Starting webpack-dev-server, port: 51420 dir: C:\Users\Johannes\Desktop\my-starter-project
   C:\Program Files\nodejs\node.exe C:\Users\Johannes\Desktop\my-starter-project\node_modules\webpack-dev-server\bin\webpack-dev-server.js --config C:\Users\Johannes\Desktop\my-starter-project\webpack.config.js --port 51420 -d --inline=false
2019-08-23 11:29:29.624  INFO 1080 --- [        webpack] dev-webpack                              : i ｢wds｣: Project is running at http://localhost:51420/webpack-dev-server/
2019-08-23 11:29:29.624  INFO 1080 --- [        webpack] dev-webpack                              : i ｢wds｣: webpack output is served from /
2019-08-23 11:29:29.624  INFO 1080 --- [        webpack] dev-webpack                              : i ｢wds｣: Content not from webpack is served from C:\Users\Johannes\Desktop\my-starter-project\target\classes\META-INF\VAADIN, src/main/webapp
2019-08-23 11:29:34.039  INFO 1080 --- [        webpack] dev-webpack                              :          Serving the 'stats.json' file dynamically.
2019-08-23 11:29:34.086  INFO 1080 --- [        webpack] dev-webpack                              : i ｢wdm｣: Hash: 690341654bcdbbe29e92
2019-08-23 11:29:34.086  INFO 1080 --- [        webpack] dev-webpack                              : Version: webpack 4.30.0
2019-08-23 11:29:34.086  INFO 1080 --- [        webpack] dev-webpack                              : Time: 4447ms
2019-08-23 11:29:34.086  INFO 1080 --- [        webpack] dev-webpack                              : Built at: 2019-08-23 11:29:34
2019-08-23 11:29:34.086  INFO 1080 --- [        webpack] dev-webpack                              :                                                                  Asset       Size      Chunks                    Chunk Names
2019-08-23 11:29:34.086  INFO 1080 --- [        webpack] dev-webpack                              :                      build/vaadin-bundle-a9542986218787df0d57.cache.js    3.5 MiB      bundle  [emitted]  [big]  bundle
2019-08-23 11:29:34.086  INFO 1080 --- [        webpack] dev-webpack                              :                  build/vaadin-bundle.es5-012aad1b579f3492d762.cache.js   3.89 MiB  bundle.es5  [emitted]  [big]  bundle.es5
2019-08-23 11:29:34.086  INFO 1080 --- [        webpack] dev-webpack                              :                                       build\webcomponentsjs\LICENSE.md   1.52 KiB              [emitted]         
2019-08-23 11:29:34.086  INFO 1080 --- [        webpack] dev-webpack                              :                                        build\webcomponentsjs\README.md   10.9 KiB              [emitted]         
2019-08-23 11:29:34.086  INFO 1080 --- [        webpack] dev-webpack                              :                      build\webcomponentsjs\bundles\webcomponents-ce.js   16.7 KiB              [emitted]         
2019-08-23 11:29:34.086  INFO 1080 --- [        webpack] dev-webpack                              :                  build\webcomponentsjs\bundles\webcomponents-ce.js.map    112 KiB              [emitted]         
2019-08-23 11:29:34.086  INFO 1080 --- [        webpack] dev-webpack                              :                build\webcomponentsjs\bundles\webcomponents-sd-ce-pf.js    108 KiB              [emitted]         
2019-08-23 11:29:34.086  INFO 1080 --- [        webpack] dev-webpack                              :            build\webcomponentsjs\bundles\webcomponents-sd-ce-pf.js.map    653 KiB              [emitted]         
2019-08-23 11:29:34.086  INFO 1080 --- [        webpack] dev-webpack                              :                   build\webcomponentsjs\bundles\webcomponents-sd-ce.js   80.9 KiB              [emitted]         
2019-08-23 11:29:34.086  INFO 1080 --- [        webpack] dev-webpack                              :               build\webcomponentsjs\bundles\webcomponents-sd-ce.js.map    527 KiB              [emitted]         
2019-08-23 11:29:34.087  INFO 1080 --- [        webpack] dev-webpack                              :                      build\webcomponentsjs\bundles\webcomponents-sd.js   65.1 KiB              [emitted]         
2019-08-23 11:29:34.087  INFO 1080 --- [        webpack] dev-webpack                              :                  build\webcomponentsjs\bundles\webcomponents-sd.js.map    415 KiB              [emitted]         
2019-08-23 11:29:34.087  INFO 1080 --- [        webpack] dev-webpack                              :                   build\webcomponentsjs\custom-elements-es5-adapter.js  924 bytes              [emitted]         
2019-08-23 11:29:34.087  INFO 1080 --- [        webpack] dev-webpack                              : build\webcomponentsjs\entrypoints\custom-elements-es5-adapter-index.js  656 bytes              [emitted]         
2019-08-23 11:29:34.087  INFO 1080 --- [        webpack] dev-webpack                              :        build\webcomponentsjs\entrypoints\webcomponents-bundle-index.js   1.66 KiB              [emitted]         
2019-08-23 11:29:34.087  INFO 1080 --- [        webpack] dev-webpack                              :            build\webcomponentsjs\entrypoints\webcomponents-ce-index.js  672 bytes              [emitted]         
2019-08-23 11:29:34.087  INFO 1080 --- [        webpack] dev-webpack                              :         build\webcomponentsjs\entrypoints\webcomponents-sd-ce-index.js  851 bytes              [emitted]         
2019-08-23 11:29:34.087  INFO 1080 --- [        webpack] dev-webpack                              :      build\webcomponentsjs\entrypoints\webcomponents-sd-ce-pf-index.js   1.19 KiB              [emitted]         
2019-08-23 11:29:34.087  INFO 1080 --- [        webpack] dev-webpack                              :            build\webcomponentsjs\entrypoints\webcomponents-sd-index.js  761 bytes              [emitted]         
2019-08-23 11:29:34.087  INFO 1080 --- [        webpack] dev-webpack                              :                                     build\webcomponentsjs\package.json      3 KiB              [emitted]         
2019-08-23 11:29:34.087  INFO 1080 --- [        webpack] dev-webpack                              :                          build\webcomponentsjs\webcomponents-bundle.js    109 KiB              [emitted]         
2019-08-23 11:29:34.087  INFO 1080 --- [        webpack] dev-webpack                              :                      build\webcomponentsjs\webcomponents-bundle.js.map    655 KiB              [emitted]         
2019-08-23 11:29:34.087  INFO 1080 --- [        webpack] dev-webpack                              :                          build\webcomponentsjs\webcomponents-loader.js   6.13 KiB              [emitted]         
2019-08-23 11:29:34.087  INFO 1080 --- [        webpack] dev-webpack                              : Entrypoint bundle [big] = build/vaadin-bundle-a9542986218787df0d57.cache.js
2019-08-23 11:29:34.087  INFO 1080 --- [        webpack] dev-webpack                              : Entrypoint bundle.es5 [big] = build/vaadin-bundle.es5-012aad1b579f3492d762.cache.js
2019-08-23 11:29:34.087  INFO 1080 --- [        webpack] dev-webpack                              : [../node_modules/@polymer/app-layout/app-drawer-layout/app-drawer-layout.js] 9.42 KiB {bundle.es5} [built]
2019-08-23 11:29:34.087  INFO 1080 --- [        webpack] dev-webpack                              : [../node_modules/@polymer/app-layout/app-drawer-layout/app-drawer-layout.js] 7.75 KiB {bundle} [built]
2019-08-23 11:29:34.087  INFO 1080 --- [        webpack] dev-webpack                              : [../node_modules/@polymer/app-layout/app-drawer/app-drawer.js] 21.5 KiB {bundle.es5} [built]
2019-08-23 11:29:34.087  INFO 1080 --- [        webpack] dev-webpack                              : [../node_modules/@polymer/app-layout/app-drawer/app-drawer.js] 18.2 KiB {bundle} [built]
2019-08-23 11:29:34.087  INFO 1080 --- [        webpack] dev-webpack                              : [../node_modules/@polymer/iron-collapse/iron-collapse.js] 7.99 KiB {bundle.es5} [built]
2019-08-23 11:29:34.087  INFO 1080 --- [        webpack] dev-webpack                              : [../node_modules/@polymer/iron-collapse/iron-collapse.js] 7.58 KiB {bundle} [built]
2019-08-23 11:29:34.087  INFO 1080 --- [        webpack] dev-webpack                              : [../node_modules/@polymer/iron-icon/iron-icon.js] 5.88 KiB {bundle.es5} [built]
2019-08-23 11:29:34.087  INFO 1080 --- [        webpack] dev-webpack                              : [../node_modules/@polymer/iron-icon/iron-icon.js] 5.56 KiB {bundle} [built]
2019-08-23 11:29:34.087  INFO 1080 --- [        webpack] dev-webpack                              : [../node_modules/@polymer/paper-badge/paper-badge.js] 9.21 KiB {bundle.es5} [built]
2019-08-23 11:29:34.087  INFO 1080 --- [        webpack] dev-webpack                              : [../node_modules/@polymer/paper-badge/paper-badge.js] 7.63 KiB {bundle} [built]
2019-08-23 11:29:34.087  INFO 1080 --- [        webpack] dev-webpack                              : [../node_modules/@polymer/paper-ripple/paper-ripple.js] 18.8 KiB {bundle.es5} [built]
2019-08-23 11:29:34.087  INFO 1080 --- [        webpack] dev-webpack                              : [../node_modules/@polymer/paper-ripple/paper-ripple.js] 18.2 KiB {bundle} [built]
2019-08-23 11:29:34.087  INFO 1080 --- [        webpack] dev-webpack                              : [../node_modules/@vaadin/flow-frontend/com/github/appreciated/app-layout/app-layout-styles-lumo.css] 5.86 KiB {bundle} {bundle.es5} [built]
2019-08-23 11:29:34.087  INFO 1080 --- [        webpack] dev-webpack                              : [../target/frontend/generated-flow-imports.js] 3.28 KiB {bundle.es5} [built]
2019-08-23 11:29:34.087  INFO 1080 --- [        webpack] dev-webpack                              : [../target/frontend/generated-flow-imports.js] 3.26 KiB {bundle} [built]
2019-08-23 11:29:34.087  INFO 1080 --- [        webpack] dev-webpack                              :     + 299 hidden modules
2019-08-23 11:29:34.087  INFO 1080 --- [        webpack] dev-webpack                              : i ｢wdm｣: Compiled successfully.
2019-08-23 11:29:34.087  INFO 1080 --- [        webpack] dev-webpack                              : 
----------------- Frontend compiled successfully. -----------------
2019-08-23 11:29:34.325  INFO 1080 --- [  restartedMain] o.s.s.concurrent.ThreadPoolTaskExecutor  : Initializing ExecutorService 'applicationTaskExecutor'
2019-08-23 11:29:34.419  INFO 1080 --- [  restartedMain] o.s.b.d.a.OptionalLiveReloadServer       : LiveReload server is running on port 35729
2019-08-23 11:29:34.512  INFO 1080 --- [  restartedMain] o.s.b.w.embedded.tomcat.TomcatWebServer  : Tomcat started on port(s): 8080 (http) with context path ''
2019-08-23 11:29:34.515  INFO 1080 --- [  restartedMain] c.github.appreciated.spring.Application  : Started Application in 65.653 seconds (JVM running for 66.4)
2019-08-23 11:30:34.220  WARN 1080 --- [alina-utility-1] com.vaadin.flow.server.VaadinSession     : A VaadinSession instance not associated to any service is getting unbound. Session destroy events will not be fired and UIs in the session will not get detached. This might happen if a session is deserialized but never used before it expires.
```
</details>
As it can be seen `apexcharts` and also `onecolor` won't be installed

### Current workarround
~A workaround is to simply create a class which extends component which contains the disired `@NpmPackage`. The dependency will then be avaiable~

### Expected behavior
Using multiple `@NpmPackage` Annotations should still cause the dependencies to be installed.

- Versions:
    - Vaadin 14.0.1
    - Java 8
    - Windows 10

--- a/vaadin-spring-tests/test-spring-common/src/main/java/com/vaadin/flow/spring/test/DoubleNpmAnnotationHolder.java
+++ b/vaadin-spring-tests/test-spring-common/src/main/java/com/vaadin/flow/spring/test/DoubleNpmAnnotationHolder.java
@@ -1,0 +1,15 @@
+package com.vaadin.flow.spring.test;
+
+import com.vaadin.flow.component.dependency.NpmPackage;
+
+/**
+ * Since @NpmPackage annotations should be discovered without a @Route
+ * annotation, they have been separated here for the scanner to find (and
+ * avoid being found some other way)
+ *
+ * @see com.vaadin.flow.spring.test.DoubleNpmAnnotationView
+ */
+@NpmPackage(value = "@polymer/paper-input", version = "3.0.2")
+@NpmPackage(value = "@polymer/paper-checkbox", version = "3.0.1")
+public class DoubleNpmAnnotationHolder {
+}

--- a/vaadin-spring-tests/test-spring-common/src/main/java/com/vaadin/flow/spring/test/DoubleNpmAnnotationHolder.java
+++ b/vaadin-spring-tests/test-spring-common/src/main/java/com/vaadin/flow/spring/test/DoubleNpmAnnotationHolder.java
@@ -1,3 +1,18 @@
+/*
+ * Copyright 2000-2017 Vaadin Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
 package com.vaadin.flow.spring.test;
 
 import com.vaadin.flow.component.dependency.NpmPackage;

--- a/vaadin-spring-tests/test-spring-common/src/main/java/com/vaadin/flow/spring/test/DoubleNpmAnnotationView.java
+++ b/vaadin-spring-tests/test-spring-common/src/main/java/com/vaadin/flow/spring/test/DoubleNpmAnnotationView.java
@@ -1,3 +1,18 @@
+/*
+ * Copyright 2000-2017 Vaadin Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
 package com.vaadin.flow.spring.test;
 
 import com.vaadin.flow.component.dependency.JsModule;

--- a/vaadin-spring-tests/test-spring-common/src/main/java/com/vaadin/flow/spring/test/DoubleNpmAnnotationView.java
+++ b/vaadin-spring-tests/test-spring-common/src/main/java/com/vaadin/flow/spring/test/DoubleNpmAnnotationView.java
@@ -1,0 +1,18 @@
+package com.vaadin.flow.spring.test;
+
+import com.vaadin.flow.component.dependency.JsModule;
+import com.vaadin.flow.component.html.Div;
+import com.vaadin.flow.dom.Element;
+import com.vaadin.flow.router.Route;
+
+@JsModule("@polymer/paper-input/paper-input.js")
+@JsModule("@polymer/paper-checkbox/paper-checkbox.js")
+@Route("double-npm-annotation")
+public class DoubleNpmAnnotationView extends Div {
+    public DoubleNpmAnnotationView() {
+        Element paperInput = new Element("paper-input");
+        Element paperCheckbox = new Element("paper-checkbox");
+
+        this.getElement().appendChild(paperInput, paperCheckbox);
+    }
+}

--- a/vaadin-spring-tests/test-spring-common/src/test/java/com/vaadin/flow/spring/test/DoubleNpmAnnotationIT.java
+++ b/vaadin-spring-tests/test-spring-common/src/test/java/com/vaadin/flow/spring/test/DoubleNpmAnnotationIT.java
@@ -1,3 +1,18 @@
+/*
+ * Copyright 2000-2017 Vaadin Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
 package com.vaadin.flow.spring.test;
 
 import org.junit.Assert;

--- a/vaadin-spring-tests/test-spring-common/src/test/java/com/vaadin/flow/spring/test/DoubleNpmAnnotationIT.java
+++ b/vaadin-spring-tests/test-spring-common/src/test/java/com/vaadin/flow/spring/test/DoubleNpmAnnotationIT.java
@@ -1,0 +1,29 @@
+package com.vaadin.flow.spring.test;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+import com.vaadin.testbench.TestBenchElement;
+
+public class DoubleNpmAnnotationIT extends AbstractSpringTest {
+
+    @Test
+    public void bothPaperWebComponentsAreLoaded() throws Exception {
+        open();
+        TestBenchElement paperCheckbox = $("paper-checkbox").first();
+        TestBenchElement paperInput = $("paper-input").first();
+
+        // check that elements are on the page
+        Assert.assertNotNull(paperCheckbox);
+        Assert.assertNotNull(paperInput);
+
+        // verify that the paper components are upgraded
+        Assert.assertNotNull(paperCheckbox.$("paper-input-container"));
+        Assert.assertNotNull(paperInput.$("checkboxContainer"));
+    }
+
+    @Override
+    protected String getTestPath() {
+        return "/double-npm-annotation";
+    }
+}

--- a/vaadin-spring-tests/test-spring-common/src/test/java/com/vaadin/flow/spring/test/DoubleNpmAnnotationIT.java
+++ b/vaadin-spring-tests/test-spring-common/src/test/java/com/vaadin/flow/spring/test/DoubleNpmAnnotationIT.java
@@ -15,6 +15,8 @@
  */
 package com.vaadin.flow.spring.test;
 
+import java.util.List;
+
 import org.junit.Assert;
 import org.junit.Test;
 
@@ -25,16 +27,22 @@ public class DoubleNpmAnnotationIT extends AbstractSpringTest {
     @Test
     public void bothPaperWebComponentsAreLoaded() throws Exception {
         open();
-        TestBenchElement paperCheckbox = $("paper-checkbox").first();
-        TestBenchElement paperInput = $("paper-input").first();
+        List<TestBenchElement> paperCheckboxes = $("paper-checkbox").all();
+        List<TestBenchElement> paperInputs = $("paper-input").all();
 
         // check that elements are on the page
-        Assert.assertNotNull(paperCheckbox);
-        Assert.assertNotNull(paperInput);
+        Assert.assertTrue("Should have found a 'paper-checkbox'",
+                paperCheckboxes.size() > 0);
+        Assert.assertTrue("Should have found a 'paper-input'",
+                paperInputs.size() > 0);
 
         // verify that the paper components are upgraded
-        Assert.assertNotNull(paperCheckbox.$("paper-input-container"));
-        Assert.assertNotNull(paperInput.$("checkboxContainer"));
+        Assert.assertNotNull(
+                "'paper-checkbox' should have had element in shadow dom",
+                paperCheckboxes.get(0).$("checkboxContainer"));
+        Assert.assertNotNull(
+                "'paper-input' should have had element in shadow dom",
+                paperInputs.get(0).$("paper-input-container"));
     }
 
     @Override

--- a/vaadin-spring/src/main/java/com/vaadin/flow/spring/VaadinServletContextInitializer.java
+++ b/vaadin-spring/src/main/java/com/vaadin/flow/spring/VaadinServletContextInitializer.java
@@ -115,7 +115,7 @@ public class VaadinServletContextInitializer
                 try {
                     List<Class<?>> routeClasses = findByAnnotation(
                             getRoutePackages(), Route.class, RouteAlias.class)
-                                    .collect(Collectors.toList());
+                            .collect(Collectors.toList());
 
                     getLogger().debug(
                             "Found {} route classes. Here is the list: {}",
@@ -152,7 +152,7 @@ public class VaadinServletContextInitializer
         }
 
         private void setAnnotatedRoutes(RouteConfiguration routeConfiguration,
-                Set<Class<? extends Component>> routes) {
+                                        Set<Class<? extends Component>> routes) {
             routeConfiguration.getHandledRegistry().clean();
             for (Class<? extends Component> navigationTarget : routes) {
                 try {
@@ -196,8 +196,8 @@ public class VaadinServletContextInitializer
 
             Stream<Class<? extends Component>> hasErrorComponents = findBySuperType(
                     getErrorParameterPackages(), HasErrorParameter.class)
-                            .filter(Component.class::isAssignableFrom)
-                            .map(clazz -> (Class<? extends Component>) clazz);
+                    .filter(Component.class::isAssignableFrom)
+                    .map(clazz -> (Class<? extends Component>) clazz);
             registry.setErrorNavigationTargets(
                     hasErrorComponents.collect(Collectors.toSet()));
         }
@@ -251,7 +251,7 @@ public class VaadinServletContextInitializer
     private class DevModeServletContextListener
             implements ServletContextListener {
 
-        @SuppressWarnings({ "unchecked", "rawtypes" })
+        @SuppressWarnings({"unchecked", "rawtypes"})
         @Override
         public void contextInitialized(ServletContextEvent event) {
 
@@ -279,7 +279,8 @@ public class VaadinServletContextInitializer
             // WebComponentExporter.class
             Set<String> allClasses = Collections.singleton("");
             Set<Class<?>> classes = findByAnnotation(allClasses, customLoader,
-                    Route.class, NpmPackage.class).collect(Collectors.toSet());
+                    Route.class, NpmPackage.class,
+                    NpmPackage.Container.class).collect(Collectors.toSet());
 
             classes.addAll(findBySuperType(allClasses, customLoader,
                     WebComponentExporter.class).collect(Collectors.toSet()));
@@ -314,7 +315,7 @@ public class VaadinServletContextInitializer
 
                 Set<Class<?>> webComponentExporters = findBySuperType(
                         getWebComponentPackages(), WebComponentExporter.class)
-                                .collect(Collectors.toSet());
+                        .collect(Collectors.toSet());
 
                 try {
                     initializer.onStartup(webComponentExporters,
@@ -344,7 +345,7 @@ public class VaadinServletContextInitializer
      * {@code context} provided.
      *
      * @param context
-     *            the application context
+     *         the application context
      */
     public VaadinServletContextInitializer(ApplicationContext context) {
         appContext = context;
@@ -420,12 +421,12 @@ public class VaadinServletContextInitializer
     }
 
     private Stream<Class<?>> findByAnnotation(Collection<String> packages,
-            Class<? extends Annotation>... annotations) {
+                                              Class<? extends Annotation>... annotations) {
         return findByAnnotation(packages, appContext, annotations);
     }
 
     private Stream<Class<?>> findByAnnotation(Collection<String> packages,
-            ResourceLoader loader, Class<? extends Annotation>... annotations) {
+                                              ResourceLoader loader, Class<? extends Annotation>... annotations) {
         ClassPathScanningCandidateComponentProvider scanner = new ClassPathScanningCandidateComponentProvider(
                 false);
 
@@ -438,12 +439,12 @@ public class VaadinServletContextInitializer
     }
 
     private Stream<Class<?>> findBySuperType(Collection<String> packages,
-            Class<?> type) {
+                                             Class<?> type) {
         return findBySuperType(packages, appContext, type);
     }
 
     private Stream<Class<?>> findBySuperType(Collection<String> packages,
-            ResourceLoader loader, Class<?> type) {
+                                             ResourceLoader loader, Class<?> type) {
         ClassPathScanningCandidateComponentProvider scanner = new ClassPathScanningCandidateComponentProvider(
                 false);
         scanner.setResourceLoader(loader);
@@ -484,7 +485,7 @@ public class VaadinServletContextInitializer
     private Collection<String> getErrorParameterPackages() {
         return Stream
                 .concat(Stream
-                        .of(HasErrorParameter.class.getPackage().getName()),
+                                .of(HasErrorParameter.class.getPackage().getName()),
                         getDefaultPackages().stream())
                 .collect(Collectors.toSet());
     }
@@ -545,7 +546,7 @@ public class VaadinServletContextInitializer
                 .collect(Collectors.toList());
 
         public CustomResourceLoader(ResourceLoader resourceLoader,
-                List<String> addedBlacklist, List<String> addedWhiteListed) {
+                                    List<String> addedBlacklist, List<String> addedWhiteListed) {
             super(resourceLoader);
 
             Objects.requireNonNull(addedBlacklist,
@@ -647,13 +648,13 @@ public class VaadinServletContextInitializer
          * Constructor.
          *
          * @param context
-         *            the ServletContext
+         *         the ServletContext
          * @param registration
-         *            the ServletRegistration for this ServletConfig instance
+         *         the ServletRegistration for this ServletConfig instance
          */
         private SpringStubServletConfig(ServletContext context,
-                ServletRegistrationBean registration,
-                ApplicationContext appContext) {
+                                        ServletRegistrationBean registration,
+                                        ApplicationContext appContext) {
             this.context = context;
             this.registration = registration;
             this.appContext = appContext;
@@ -698,11 +699,11 @@ public class VaadinServletContextInitializer
          * Creates a DeploymentConfiguration.
          *
          * @param context
-         *            the ServletContext
+         *         the ServletContext
          * @param registration
-         *            the ServletRegistrationBean to get servlet parameters from
+         *         the ServletRegistrationBean to get servlet parameters from
          * @param servletClass
-         *            the class to look for properties defined with annotations
+         *         the class to look for properties defined with annotations
          * @return a DeploymentConfiguration instance
          */
         public static DeploymentConfiguration createDeploymentConfiguration(

